FindMe/Location/LocationManager.m

```diff
@@ -69,10 +69,15 @@ - (void)beginTracking
         [self.locationManager startUpdatingLocation];
     }
     else {
-        
+        [self requestLocationPermission];
     }
 }
 
+- (void)stopTracking
+{
+    [self.locationManager stopUpdatingLocation];
```

Comment:
What happens if it’s already stopped? Don’t know the answer. Just curious.

Reply:
the code is designed in such a way that this function can only be called when it's already tracking